### PR TITLE
Add content in share process

### PR DESCRIPTION
**Pull request by aledeg**
I've recently discovered that we can transmit the description directly
to Shaarli. So I modified the sharing process to include the content
in the sharing link.

### CODE REVIEW

**Comment by Frenzie** [comment]
Can there be a danger that it'll be too long unless cut off at some point?

**Comment by aledeg** [comment]
Possibly

**Comment by Frenzie** [comment]
Then I propose a maxlength substring or equivalent. ;-)

**Comment by aledeg** [comment]
I'll modify this PR to include that kind of feature.

File: app/views/helpers/index/normal/entry_bottom.phtml [hunk 1]
```diff
@@ -87,6 +87,7 @@
 						}
 						$share_options['link'] = $link;
 						$share_options['title'] = $title;
+						$share_options['content'] = $this->entry->content();
```
Comment (Alkarex) [inline app/views/helpers/index/normal/entry_bottom.phtml:90]: Hum, we have a risk of duplicating the article content many times here, right?
I think this should be replaced by a dynamic menu, either server side or client side

Comment (aledeg) [reply app/views/helpers/index/normal/entry_bottom.phtml:90]: I am not sure I follow your reasoning. Could you explain it?

Comment (Alkarex) [reply app/views/helpers/index/normal/entry_bottom.phtml:90]: Yes: I am worried about the output size. If I am not mistaken, the HTML output will contain up to several times the content of each article, considerably increasing the size of the page

Comment (aledeg) [reply app/views/helpers/index/normal/entry_bottom.phtml:90]: Ok, I understand all the concerns. I think it's better if we keep it the way it is. I am closing this PR.

Comment (Alkarex) [reply app/views/helpers/index/normal/entry_bottom.phtml:90]: Ok. Do you see any problem with a dynamic menu (either client side or server side)?

Comment (aledeg) [reply app/views/helpers/index/normal/entry_bottom.phtml:90]: I think that if one want the content, he will copy and paste it as I am doing at the moment. I don't think that spending time on that is necessary.

Comment (Alkarex) [reply app/views/helpers/index/normal/entry_bottom.phtml:90]: Ok

**Pull request closed without merging**